### ISSUE TITLE
Review Product Backlog

### ISSUE THREAD

**Issue by FabianWilms** [WeisswurstSystems/WWM-project#1]
Es sollten zumindest für Milestone 1 alle Use-Cases zuinächst einmal grob dokumentiert sein. Um die detailierte Beschreibung kümmere ich mich wenn alle UCs drin sind.

@peter-mueller 
@darenegade 

Wenn ihr Zeit habt mal drüber schauen und ergänzen was ihr noch für wichtig/sinnvoll haltet. Ich frag am Montag mal in die Runde was die anderen USER so denken :P

**Comment by peter-mueller** [comment]
@FabianWilms

@xdoo soll uns eine Event storming Demonstration dazu machen :smile:

**Comment by darenegade** [comment]
Unbedingt notwendig bei dieser komplexen Domäne. 😂
Aber würde mich wundern, wenn Claus das bei it@m schon mal eingesetzt hat und besonders lustig ist Event storming als Demonstration. Steht er alleine vorne und wir schauen ihm dabei zu😅😂

**Comment by xdoo** [comment]
Nein. In  diesem Fall würde ich vorne stehen und ihr stürmt :) Können wir gerne machen @peter-mueller
